### DISCUSSION TITLE
Core Body Sensor Integration

### DISCUSSION THREAD

**Discussion by supreik** [dvmarinoff/Auuki#179] [Ideas]
Just as an Idea because there is already Moxy integrated. I've seen there is an documentaton available: 
https://corebodytemp.com/pages/tech-core-developer-information

With this specific sensor it's possible to monitor the body temp (= body temp threshold). Could be interesting during indoor session. I'm able to get a device so I can be a testuser.

**Comment by dvmarinoff** [comment]
Took a glance at the docs. Everything seems pretty standard, maybe a 10-12 hour job to integrate it. I'll let you know when I find some free time to added. Maybe this or next weekend.

**Comment by supreik** [comment]
@dvmarinoff :-) Did you find some time to integrate Core Sensore to the project?

**Comment by dvmarinoff** [comment]
@supreik Sorry, not yet! Just working on the Zwift Click integration, which turns out to be a bit frustrating. Will definitely try to find some time to do the Core Temp sensor. I'll let you know when I have something ready for testing.

**Comment by dvmarinoff** [comment]
@supreik Hey, I've added support for the Core Temp sensor to Flux. It's currently on the development version of the app on: [https://flux-dev.vercel.app/](https://flux-dev.vercel.app/).

I've tested it only with a bluetooth simulator. It's ready for testing with the real device if your offer still stands.
Here is how it looks now:

<img width="320" src="https://github.com/dvmarinoff/Flux/assets/7523832/3908d526-866d-489a-8404-796cb2b02d07" />
<img width="320" src="https://github.com/dvmarinoff/Flux/assets/7523832/3a85a839-a0f1-4738-a106-5b86e02f11e8" />

**Reply by dvmarinoff** [reply]
Forgot to mention that the output fields for the values are on the second tab in the UI. Flux can also read Heart Rate if it's proxied from the Core Temp sensor, but you need to trigger the that option from the settings menu for Core Temp.

**Comment by dvmarinoff** [comment]
Just added Smo2, Thb, Core and Skin temperature to the ride report data:

<img width="287" alt="Screenshot 2024-03-16 at 12 49 47" src="https://github.com/dvmarinoff/Flux/assets/7523832/839ee279-87ca-4ab9-aea6-f9edf862ebb9">
